core/app/[locale]/(default)/cart/page-data.ts

```diff
@@ -262,18 +264,34 @@ export const getCart = async (variables: Variables) => {
   return data;
 };
 
-export const getShippingCountries = async (geography: FragmentOf<typeof GeographyFragment>) => {
-  const hasAccessToken = Boolean(process.env.BIGCOMMERCE_ACCESS_TOKEN);
-  const shippingZones = hasAccessToken ? await getShippingZones() : [];
-  const countries = geography.countries ?? [];
-
-  const uniqueCountryZones = new Set(
-    shippingZones.map((zone) => zone.locations.map((location) => location.country_iso2)).flat(),
-  );
-
-  return countries.filter((countryDetails) => {
-    const isCountryInTheList = uniqueCountryZones.has(countryDetails.code);
+const SupportedShippingDestinationsQuery = graphql(`
+  query SupportedShippingDestinations {
+    site {
+      settings {
+        shipping {
+          supportedShippingDestinations {
+            countries {
+              entityId
+              code
+              name
+              statesOrProvinces {
+                entityId
+                name
+                abbreviation
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+`);
 
-    return isCountryInTheList || !hasAccessToken;
+export const getShippingCountries = cache(async () => {
+  const { data } = await client.fetch({
+    document: SupportedShippingDestinationsQuery,
+    fetchOptions: { next: { revalidate } },
```

Comment:
Is this the right caching strategy? I wasn't sure if CAT influences the returned list of countries 🤔